update/index.mjs

```diff
@@ -36,18 +37,20 @@ export const run = async ({ source }) => {
 
     report.success(`found version ${version}`);
 
-    if (version === currentVersion) {
+    if (version !== currentVersion) {
```

Comment:
this was a bug which is fixed in https://github.com/amir-s/dev/commit/33a856f438ce17d9737b6a08071da1583d4934d8